base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -1255,13 +1256,21 @@ where TBackend: OutputManagerBackend + 'static
                     .map(|v| (v, output.features, output.script, output.script_offset_public_key))
             })
             .map(|(output, features, script, script_offset_public_key)| {
+                let beta_hash = Blake256::new()
+                    .chain(script.as_bytes())
+                    .chain(features.to_bytes())
+                    .chain(script_offset_public_key.as_bytes())
```

Comment:
Not sure if this is still necessary to keep in

Reply:
Yip, maybe not.

Reply:
But this will enhance the contract between Alice and Bob